app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -37,6 +37,10 @@ def healthcheck
 
     private
 
+    def skip_sentry_exception_types
+      ApplicationController::SKIP_SENTRY_EXCEPTION_TYPES + [Common::Exceptions::GatewayTimeout]
```

Comment:
I'm trying to think of pros and cons of changing this to `super + [Common::Exceptions::GatewayTimeout]`?    Will `ApplicationController#skip_sentry_exception_types` always contain the minimum array of "skip sentry exception types"?

Also, it seems we currently now have two ways of overwriting `skip_sentry_exception_types`.  There is a `concern` (`IgnoreNotFound`) that is included in `EVSSClaimsController` (and nowhere else).  And here we just overwrite the method in the controller.  Should this also be made a concern?  Or should `skip_sentry_exception_types` just be defined in `EVSSClaimsController`?